Sources/VLCSettingsController.m

```diff
@@ -35,19 +46,33 @@ - (instancetype)initWithStyle:(UITableViewStyle)style
 
 - (void)viewDidLoad
 {
+    [super viewDidLoad];
+
     self.modalPresentationStyle = UIModalPresentationFormSheet;
     self.delegate = self;
     self.showDoneButton = NO;
     self.showCreditsFooter = NO;
+    self.neverShowPrivacySettings = YES;
+    
+    self.navigationItem.leftBarButtonItem = [[UIBarButtonItem alloc] initWithTitle:NSLocalizedString(@"BUTTON_ABOUT", nil) style:UIBarButtonItemStylePlain target:self action:@selector(pushAboutViewController:)];
+    self.navigationItem.leftBarButtonItem.accessibilityIdentifier = VLCAccessibilityIdentifier.about;
+    
+    self.tableView.estimatedRowHeight = MINIMUM_SETTINGS_ROW_HEIGHT;
+    self.tableView.separatorStyle = UITableViewCellSeparatorStyleNone;
+    
     [self themeDidChange];
 }
 
 - (void)themeDidChange
 {
-    self.view.backgroundColor = PresentationTheme.current.colors.settingsBackground;
-    self.tableView.separatorColor = PresentationTheme.current.colors.settingsSeparatorColor;
+    self.view.backgroundColor = PresentationTheme.current.colors.background;
     [self.tableView reloadData];
     [self setNeedsStatusBarAppearanceUpdate];
+    
+    // Force UIKit to reload after changing UIAppearance
+    UIView *superView = self.tableView.superview;
+    [self.tableView removeFromSuperview];
+    [superView addSubview:self.tableView];
```

Comment:
This looks wrong to me and like fixing a symptom instead of the underlying issue, which is that views don't reload after the appearance change. This would be a general problem that should be fixed at a central point or if we can't fix this we should think about not defining certain properties in UIAppearance

Reply:
It's been fixed with a `reloadSections`